community/events.html

```diff
@@ -371,11 +371,22 @@ <h5 style="margin-bottom: 18px;">Ballerina: A Language to Develop, Consume and C
                     </td>	
                     <td class="cEventURL"><a class="cEventRegistration" href="https://www.meetup.com/sfjava/events/284669431" target="_blank" style="text-align: center;">More info</a></td>	
                 </tr>
-                
-                
-                
-                
-                
+                 <tr class="event-expiry"  data-expiry="March 24, 2022 19:00:00">	
+                    <td class="cEventDateContainer cEventDateContainer1">	
+                       <p class="cEventDate1 cEventDateNum dateDisplay" style="margin-bottom: 15px;">June 8, 2022</p>	
+                       <p class="cEventDate1 dateDisplay" style="margin-bottom: 15px">Wed., 6.10 a.m. EDT</p>	
+                       <p class="cEventLocation" style="text-align: right;">Online</p>	
+                    </td>	
+                    <td class="cEventDetail" id="eventDetails" >	
+                       <a target="_blank" href="https://www.womentech.net/speaker/Anupama/Pathirage/70302">	
+                          <p class="cEventName" style="margin-top: -1px;font-size: 18px;">Meetup</p>	
+                       </a>	
+                       <h5 style="margin-bottom: 18px;">Ballerina: Cloud-Native Middleware as a Programming Language</h5>		
+                       <a style="color: #20b6b0 ;font-weight:400;font-size: 16px;" href="https://twitter.com/sameerajayasoma">Anupama Pathirage</a>, Director of Engineering, WSO2
```

Comment:
Incorrect Twitter link

Reply:
This is fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/4321/.